Onboarding new team member (#146)
Openscapes/approach-guide: approach/onboarding.md

title: Onboarding
---

## Welcome to the team!

Through time, Openscapes has been onboarding folks for different roles and purposes. We are excited for more communities to be able to use the Openscapes approach!

A common way we start collaborating with folks is to to deliver the Champions Program, and this page was first built to document the onboarding process to contribute to a Champions Cohort. This focused on onboarding folks to collaborate in different capacities — i.e. as mentors, assistants, evaluators, lesson contributors, guest teachers — so we will be trying to be as streamlined as possible with the most relevant information and skill-building to meet specific needs and timeframes.

To support the growing Openscapes work, we are also onboarding core team members to support Mentors, build and maintain infrastructure, as well as leading Champions Cohorts and other events. Here is some of the ways we've approached this as well.

::: {.callout-note icon="false"}
## Onboarding Issue: NASA Openscapes Cloud Team Member

> We'll soon be onboarding a new team member, whose primary focus is to support the NASA Openscapes Mentor community! From our job posting for this 1/2 time contracting position, we had 24 candidates apply. Working with @lizneeley of [Liminal](https://www.liminalcreations.com/), who is an expert in science communication, design, and sensemaking, we developed a hiring rubric and designed an interview process to create an equitable hiring and evaluation process that aligned with our open kinder science values, community, timeline, and that we can reuse and improve upon in the future. In February we led 2 rounds of interviews with really stellar candidates. We are so grateful for these candidates working in this space with very diverse skillsets and backgrounds who are interested in this position and being a part of this NASA Openscapes work and the open science community.

Reviewer: ```suggestion
We'll soon be onboarding a new team member, whose primary focus is to support the NASA Openscapes Mentor community! From our job posting for this 1/2 time contracting position, we had 24 candidates apply. Working with @lizneeley of [Liminal](https://www.liminalcreations.com/), who is an expert in science communication, design, and sensemaking, we developed a hiring rubric and designed an interview process to create an equitable hiring and evaluation process that aligned with our open kinder science values, community, timeline, and that we can reuse and improve upon in the future. In February 2024 we led 2 rounds of interviews with really stellar candidates. We are so grateful for these candidates working in this space with very diverse skillsets and backgrounds who are interested in this position and being a part of this NASA Openscapes work and the open science community.
```

These are specifics about the role that we shared before the final round of interviews -

## Background: Openscapes Cloud Team Member

> **Ethos:** The Openscapes Cloud Team Member position will be supporting NASA Openscapes Mentors and part of the Openscapes Core Team. We expect this to be ¾ NASA and 1⁄4 Core. This isn’t merely a contract; the Openscapes team are leaders in the open science movement. Collaboration is essential to this job: you will not only complete tasks assigned to you, but you will lead and role-model how to support others in Cloud/open source software. To do that, you will need to identify and then support the needs of the NASA Mentors and the Openscapes broader community: building trust and being comfortable learning with others will be essential.

Reviewer: ```suggestion
**Ethos:** The Openscapes Cloud Team Member position will be supporting NASA Openscapes Mentors and be part of the Openscapes Core Team. We expect this to be ¾ NASA and 1⁄4 Core. This isn’t merely a contract; the Openscapes team are leaders in the open science movement. Collaboration is essential to this job: you will not only complete tasks assigned to you, but you will lead and role-model how to support others in Cloud/open source software. To do that, you will need to identify and then support the needs of the NASA Mentors and the Openscapes broader community: building trust and being comfortable learning with others will be essential.
```

## Onboarding

Onboarding the Cloud Team Member will follow the way we have grown Openscapes teams so far: using our [Flywheel](https://openscapes.org/approach#openscapes-flywheel) ([Robinson & Lowndes 2022](https://eartharxiv.org/repository/view/4560/)). Our approach is "onboard and delegate" and not "train the trainers". The new team member will join existing calls with the Core team and NASA Openscapes Mentors, where they will first listen and learn, understanding the culture and needs first as they build trust and then offer advice and skills to support the Mentors. We'll be working closely with this person to help identify places to contribute and scoping priorities identified by the Mentors to support NASA Earthdata users. We'll all have a lot to learn from each other!

We have been documenting onboarding from previous turns of the flywheel, and these resources will help us and this new person get up to speed and contribute - including by helping improve the onboarding documentation.

![Image](https://github.com/Openscapes/how_we_work/assets/5891909/80520158-b44e-4309-9e09-6c2c577c7bbe)

We're really excited to welcome this person to the community and work together ❤
:::

> The above was posted as "How we work" [issue #323](https://github.com/Openscapes/how_we_work/issues/323).

Reviewer: ```suggestion
The above was posted as [issue #323](https://github.com/Openscapes/how_we_work/issues/323) in our "How we work" repo.
```

Your weekly synchronous responsibilities will include: 

-   **Openscapes Core Team meetings** - (1.5 hrs/week - currently Tuesdays 8:30-9:45am PT)

    -   Purpose: onboarding, sharing across the Openscapes Core Team.

-   **NASA Mentor Calls and/or Coworking** (1.5 hrs/alternating Wednesdays and Thursdays 10-11:30 PT)

    -   Purpose: get to know Mentors, how we work, contribute to the Cookbook structure and lessons

    -   Focus: Cookbook hackdays, focus on contributing and how to onboard new Mentors 

-   **Suborbital Coworking - to be scheduled, likely weekly, to support teams and to we design the plan and prepare for the Notebook Clinic**\

In addition, there are additional synchronous calls upcoming this spring:

-   **NASA Openscapes Champions Cohort** (1.5 hrs/week for 10 weeks, to be scheduled likely in Apr-Jun)

> -   **Notebook Clinic for Suborbital teams** (1.5 hrs/week for 8-10 weeks, to be scheduled likely in the fall)

Reviewer 2: ```suggestion
In addition, there are additional synchronous calls upcoming this year:

-   **NASA Openscapes Champions Cohort** (1.5 hrs/week for 10 weeks, to be scheduled likely in the Fall)

-   **Notebook Clinic for Suborbital teams** (structure to be determined, to be scheduled likely in the Fall)
```
